jujugui/static/gui/src/app/components/added-services-list/label/test-label.js

```diff
@@ -60,10 +72,13 @@ describe('AddedServicesLabel', () => {
       changeState,
       bundleURL: 'cs:bundle/wiki-simple-4'
     });
-    component.find('.inspector-view__label-link-list li').at(1).simulate('click');
+    component
+      .find('.inspector-view__label-link-list li')
+      .at(1)
+      .simulate('click');
     assert.equal(changeState.callCount, 1);
-    assert.deepEqual(
-      changeState.args[0][0], {postDeploymentPanel: 'wiki-simple/bundle/4'});
+    assert.deepEqual(changeState.args[0][0], {
+      postDeploymentPanel: 'wiki-simple/bundle/4'
```

Comment:
Prettier is making some choices here that I personally find less legible, much like this choice for wrapping.

Reply:
Sure, I'd probably agree in this case. There are other parts I wouldn't personally choose either but in my opinion, the ability to completely delegate formatting to a computer and never have to go-round and fix a linting error that broke a build when I'm already at dinner far outweighs these personal preferences that I'll invariably forget I even ad in a couple of weeks.

Reply:
It's a person who has to read this in the future so prioritizing consistency and legibility is a priority

Reply:
Consistency is a major concern with Prettier. This always for no inconsistency to creep in over time that does not match a linting rule. Personally I feel this code is no less legible but of course that is a personal preference.

Reply:
But it's not consistent within itself...

Compare this example to the one here: https://github.com/juju/juju-gui/pull/3864/files/55ffb22776559cbe5947331352b95bc46a38f245#diff-c92c703841e3a00e72786a3a0ad5cf9bR100

On this line it moved the first argument onto the line with the function call, but in the link it has moved it below...

Reply:
Can you link to the line please?

Reply:
That is....I don't think GH handles 60k line diffs well ;)

scroll down and see for where I tell prettier to stay out of my nodejs ;) in transpile.js

Reply:
Interesting, I can't explain why they are different. But should we care to that level?

Reply:
> Sure, I'd probably agree in this case. There are other parts I wouldn't personally choose either but in my opinion, the ability to completely delegate formatting to a computer and never have to go-round and fix a linting error that broke a build when I'm already at dinner far outweighs these personal preferences that I'll invariably forget I even had in a couple of weeks.

That's a little dismissive, isn't it? 

I'm seeing multiple places @hatched has highlighted where I feel like the code has been made less legible--and for me legibility is a primary concern. I don't care about random formatting preferences, tbh, but when code is actually getting harder to read I don't value the time tradeoff of automatic formatting.

I agree that I don't want to argue over styling stuff, but much of what I'm seeing doesn't lead to arbitrary code style, it's doing stuff that again, I find harder to read. And like, 90% of what I'm doing in the gui is reading the code.

I still don't really care here--if the consensus is go with prettier, fine. But I'm not won over that this is *positive* change.

Reply:
@jaycee It was in no way intended to be dismissive which is why I was careful to state it was only my opinion but I'm sorry if you felt it was dismissive or that's how it read.

If you've reviewed the proposal and you're not won over that this is a positive change, then for the purposes of this exercise you're a nay. Thanks for taking the time to review.

Reply:
@hatched @anthonydillon Prettier is not inconsistent - it respects the print width. Here is the two examples cited side-by-side at char width 95: https://bit.ly/2QcOjFf - they are identical.

(...which make me wonder if this code was formatted with the default setting of 80)

Reply:
> @jaycee It was in no way intended to be dismissive which is why I was careful to state it was only my opinion but I'm sorry if you felt it was dismissive or that's how it read.
> 
> If you've reviewed the proposal and you're not won over that this is a positive change, then for the purposes of this exercise you're a nay. Thanks for taking the time to review.

I think I slightly misread your original reply, so I'll apologize for taking offense. :stuck_out_tongue_closed_eyes: 

I'm actually still going through this; it's quite a large diff, and the things that stick out are obviously the bits I disagree with. I'm trying to go through and find the bits I like.

Reply:
> @hatched @anthonydillon Prettier is not inconsistent - it respects the print width. Here is the two examples cited side-by-side at char width 95: https://bit.ly/2QcOjFf - they are identical.

Ok, I actually am happier looking at this set. Thanks for providing that example @barrymcgee